python/zfpy.pxd

```diff
@@ -31,7 +31,11 @@ cdef extern from "zfp.h":
         ptrdiff_t sx, sy, sz, sw
         void* data
     ctypedef struct zfp_stream:
-        pass
+        cython.uint minbits      
```

Comment:
I would advise against exposing `zfp_stream`, whose definition here does not match the C definition and is certain to change in future versions.  A better solution is to query these parameters using `zfp_stream_params()`.

Reply:
done